lib/phoenix_live_view/static.ex

```diff
@@ -29,22 +29,44 @@ defmodule Phoenix.LiveView.Static do
 
   ## Examples
 
-      iex> verify_session(AppWeb.Endpoint, encoded_token, static_token)
-      {:ok, %{} = decoded_session}
+      iex> verify_session(AppWeb.Endpoint, "topic", encoded_token, static_token)
+      {:ok, %Session{} = decoded_session}
 
-      iex> verify_session(AppWeb.Endpoint, "bad token", "bac static")
+      iex> verify_session(AppWeb.Endpoint, "topic", "bad token", "bac static")
       {:error, :invalid}
 
-      iex> verify_session(AppWeb.Endpoint, "expired", "expired static")
+      iex> verify_session(AppWeb.Endpoint, "topic", "expired", "expired static")
       {:error, :expired}
   """
-  def verify_session(endpoint, session_token, static_token) do
-    with {:ok, %{id: id} = session} <- verify_token(endpoint, session_token),
+  def verify_session(endpoint, topic, session_token, static_token) do
```

Comment:
Should we move this to the new session module? 🤔

Reply:
We still need to unpack the static token and merge it with the session, verify the other general tokens, etc, so we'll be tightly coupled to static if we extra to Session, so I'm unsure.

Reply:
As you prefer, but I _think_ the module was originally called static not because of the static token, but because it did the static (dead) rendering.

Reply:
sorry, you're right it's definitely about static rendering. I thought we were doing other token verification in the static module, but we are only signing, so I can extract cleanly 👍